db/schema.rb

```diff
@@ -969,8 +969,8 @@
     t.integer  "user_id"
     t.integer  "work_id"
     t.datetime "created_at"
-    t.datetime "last_viewed"
-    t.integer  "view_count",         :default => 0
+    t.datetime "last_visited"
```

Comment:
Is it OK to rename these only in the UI? If you rename the database columns you would need to setup a migration.

Reply:
sure; it'd be much easier just to make the ui change. i was just worried about it causing confusion in the future if someone wants to make a change and there's a disconnect between them

Reply:
The disconnect is fine -- column/variable/method/etc names don't need to correspond perfectly to what users see.

Also, I think you might need to update your master branch. Even though this pull request is new, it has merge conflicts, including changes to files that have been deleted.